<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 2.4.0">
  <meta name="generator" content="Hugo 0.46" />
  <meta name="author" content="Dr. rer. nat. Alexander Peltzer">

  
  
  
  
  <meta name="description" content="Bioinformatics research scientist">

  
  <link rel="alternate" hreflang="en-us" href="https://apeltzer.github.io/publication/">

  


  

  
  
  
  <meta name="theme-color" content="#3f51b5">
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">

    
    
    
      
    
    

    

    

  

  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Montserrat:400,700%7cRoboto:400,400italic,700%7cRoboto&#43;Mono">
  

  <link rel="stylesheet" href="/styles.css">
  

  
  
  

  
  <link rel="alternate" href="https://apeltzer.github.io/publication/index.xml" type="application/rss+xml" title="Alex Peltzer&#39;s webpage">
  <link rel="feed" href="https://apeltzer.github.io/publication/index.xml" type="application/rss+xml" title="Alex Peltzer&#39;s webpage">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="https://apeltzer.github.io/publication/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="twitter:site" content="@alex_peltzer">
  <meta property="twitter:creator" content="@alex_peltzer">
  
  <meta property="og:site_name" content="Alex Peltzer&#39;s webpage">
  <meta property="og:url" content="https://apeltzer.github.io/publication/">
  <meta property="og:title" content="Publications | Alex Peltzer&#39;s webpage">
  <meta property="og:description" content="Bioinformatics research scientist">
  <meta property="og:locale" content="en-us">
  
  <meta property="og:updated_time" content="2017-01-01T00:00:00&#43;01:00">
  

  

  

  <title>Publications | Alex Peltzer&#39;s webpage</title>

</head>
<body id="top" data-spy="scroll" data-target="#toc" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="/">Alex Peltzer&#39;s webpage</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      
      <ul class="nav navbar-nav navbar-right">
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#about">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#publications_selected">
            
            <span>Publications</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#posts">
            
            <span>Posts</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#projects">
            
            <span>Projects</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#talks">
            
            <span>Talks</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        
      

      
      </ul>

    </div>
  </div>
</nav>





<div class="container">
  <div class="row">
    <div class="col-md-12">
      <h1>Publications</h1>

      

      
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      

      <p>
        Filter by type:

        <select class="pub-filters pubtype-select" data-filter-group="pubtype">
          <option value="*">All</option>
          
          <option value=".pubtype-2">
            Journal article
          </option>
          
        </select>

        <select class="pub-filters" data-filter-group="year">
          <option value="*">All</option>
          
          
          
          <option value=".year-2018">
            2018
          </option>
          
          <option value=".year-2017">
            2017
          </option>
          
          <option value=".year-2016">
            2016
          </option>
          
          <option value=".year-2014">
            2014
          </option>
          
          
        </select>
      </p>

      <div id="container-publications">
        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-2 year-2018">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="https://apeltzer.github.io/publication/08-baehr/" itemprop="url">Inferring genetic origins and phenotypic traits of George Bähr, the architect of the Dresden Frauenkirche</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    For historic individuals, the outward appearance and other phenotypic characteristics remain often non-resolved. Unfortunately, images or detailed written sources are only scarcely available in many cases. Attempts to study historic individuals with genetic data so far focused on hypervariable regions of mitochondrial DNA and to some extent on complete mitochondrial genomes. To elucidate the potential of in-solution based genome-wide SNP capture methods - as now widely applied in population genetics - we extracted DNA from the 17th century remains of George Bähr, the architect of the Dresdner Frauenkirche. We were able to identify the remains to be of male origin, showing sufficient DNA damage, deriving from a single person and being thus likely authentic. Furthermore, we were able to show that George Bähr had light skin pigmentation and most likely brown eyes. His genomic DNA furthermore points to a Central European origin. We see this analysis as an example to demonstrate the prospects that new in-solution SNP capture methods can provide for historic cases of forensic interest, using methods well established in ancient DNA (aDNA) research and population genetics.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Peltzer et al
    
  </div>

  <div class="pub-publication">
    
    In <em>Scientific Reports</em>,
    
    2018
  </div>

  <div class="pub-links">
    
















<a class="btn btn-primary btn-outline btn-xs" href="http://dx.doi.org/10.1038/s41598-018-20180-z" target="_blank" rel="noopener">
  Source Document
</a>



  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-2 year-2017">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="https://apeltzer.github.io/publication/07-african/" itemprop="url">Reconstructing Prehistoric African Population Structure</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    We assembled genome-wide data from 16 prehistoric Africans. We show that the anciently divergent lineage that comprises the primary ancestry of the southern African San had a wider distribution in the past, contributing ~<sup>2</sup>&frasl;<sub>3</sub> of the ancestry of Malawi hunter-gatherers ~8100-2500 years ago, and ~<sup>1</sup>&frasl;<sub>3</sub> of Tanzanian hunter-gatherers ~1400 years ago. We document how the spread of farmers from western Africa involved complete replacement of local hunter-gatherers in some regions, and we track the spread of herders by showing that the population of a ~3100 year-old pastoralist from Tanzania contributed ancestry to people from northeast to southern Africa, including a ~1200-year-old southern African pastoralist. The deepest diversifications of African lineages were complex, involving long-distance gene flow, or a lineage more deeply diverging than that of the San contributing more to some western Africans than others. We finally leverage ancient genomes to document episodes of natural selection in southern African populations.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Skoglund et al
    
  </div>

  <div class="pub-publication">
    
    In <em>Cell</em>,
    
    2017
  </div>

  <div class="pub-links">
    
















<a class="btn btn-primary btn-outline btn-xs" href="http://dx.doi.org/10.1016/j.cell.2017.08.049%20" target="_blank" rel="noopener">
  Source Document
</a>



  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-2 year-2017">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="https://apeltzer.github.io/publication/06-greeks/" itemprop="url">Genetic origins of the Minoans and Mycenaeans</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    The origins of the Bronze Age Minoan and Mycenaean cultures have puzzled archaeologists for more than a century. We have assembled genome-wide data from 19 ancient individuals, including Minoans from Crete, Mycenaeans from mainland Greece, and their eastern neighbours from southwestern Anatolia. Here we show that Minoans and Mycenaeans were genetically similar, having at least three-quarters of their ancestry from the first Neolithic farmers of western Anatolia and the Aegean, and most of the remainder from ancient populations related to those of the Caucasus and Iran. However, the Mycenaeans differed from Minoans in deriving additional ancestry from an ultimate source related to the hunter-gatherers of eastern Europe and Siberia, introduced via a proximal source related to the inhabitants of either the Eurasian steppe or Armenia. Modern Greeks resemble the Mycenaeans, but with some additional dilution of the Early Neolithic ancestry. Our results support the idea of continuity but not isolation in the history of populations of the Aegean, before and after the time of its earliest civilizations.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Lazaridis et al
    
  </div>

  <div class="pub-publication">
    
    In <em>Nature</em>,
    
    2017
  </div>

  <div class="pub-links">
    
















<a class="btn btn-primary btn-outline btn-xs" href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature23310.html" target="_blank" rel="noopener">
  Source Document
</a>



  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-2 year-2017">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="https://apeltzer.github.io/publication/05-mummies/" itemprop="url">Ancient Egyptian mummy genomes suggest an increase of Sub-Saharan African ancestry in post-Roman periods</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Egypt, located on the isthmus of Africa, is an ideal region to study historical population dynamics due to its geographic location and documented interactions with ancient civilizations in Africa, Asia, and Europe. Particularly, in the first millennium BCE Egypt endured foreign domination leading to growing numbers of foreigners living within its borders possibly contributing genetically to the local population. Here we mtDNA and nuclear DNA from mummified humans recovered from Middle Egypt that span around 1,300 years of ancient Egyptian history from the Third Intermediate to the Roman Period. Our analyses reveal that ancient Egyptians shared more Near Eastern ancestry than present-day Egyptians, who received additional Sub-Saharan admixture in more recent times. This analysis establishes ancient Egyptian mummies as a genetic source to study ancient human history and offers the perspective of deciphering Egypt’s past at a genome-wide level.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Schuenemann &amp; Peltzer et al
    
  </div>

  <div class="pub-publication">
    
    In <em>Nature Communications</em>,
    
    2017
  </div>

  <div class="pub-links">
    
















<a class="btn btn-primary btn-outline btn-xs" href="https://www.nature.com/articles/ncomms15694" target="_blank" rel="noopener">
  Source Document
</a>



  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-2 year-2016">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="https://apeltzer.github.io/publication/04-syphilis/" itemprop="url">Origin of modern syphilis and emergence of a pandemic Treponema pallidum cluster</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    The abrupt onslaught of the syphilis pandemic that started in the late fifteenth century established this devastating infectious disease as one of the most feared in human history1. Surprisingly, despite the availability of effective antibiotic treatment since the mid-twentieth century, this bacterial infection, which is caused by Treponema pallidum subsp. pallidum (TPA), has been re-emerging globally in the last few decades with an estimated 10.6 million cases in 2008 (ref. 2).
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Arora et al
    
  </div>

  <div class="pub-publication">
    
    In <em>Nature Microbiology</em>,
    
    2016
  </div>

  <div class="pub-links">
    
















<a class="btn btn-primary btn-outline btn-xs" href="http://www.nature.com/articles/nmicrobiol2016245/" target="_blank" rel="noopener">
  Source Document
</a>



  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-2 year-2016">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="https://apeltzer.github.io/publication/03-iceage/" itemprop="url">The genetic history of Ice Age Europe</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    Modern humans arrived in Europe ~45,000 years ago, but little is known about their genetic composition before the start of farming ~8,500 years ago. Here we analyse genome-wide data from 51 Eurasians from ~45,000–7,000 years ago. Over this time, the proportion of Neanderthal DNA decreased from 3–6% to around 2%, consistent with natural selection against Neanderthal variants in modern humans. Whereas there is no evidence of the earliest modern humans in Europe contributing to the genetic composition of present-day Europeans, all individuals between ~37,000 and ~14,000 years ago descended from a single founder population which forms part of the ancestry of present-day Europeans.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Fu et al
    
  </div>

  <div class="pub-publication">
    
    In <em>Nature</em>,
    
    2016
  </div>

  <div class="pub-links">
    
















<a class="btn btn-primary btn-outline btn-xs" href="http://www.nature.com/nature/journal/v534/n7606/full/nature17993.html" target="_blank" rel="noopener">
  Source Document
</a>



  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-2 year-2016">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="https://apeltzer.github.io/publication/02-eager/" itemprop="url">EAGER: efficient ancient genome reconstruction</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    The automated reconstruction of genome sequences in ancient genome analysis is a multifaceted process. Here we introduce EAGER, a time-efficient pipeline, which greatly simplifies the analysis of large-scale genomic data sets. EAGER provides features to preprocess, map, authenticate, and assess the quality of ancient DNA samples. Additionally, EAGER comprises tools to genotype samples to discover, filter, and analyze variants. EAGER encompasses both state-of-the-art tools for each step as well as new complementary tools tailored for ancient DNA data within a single integrated solution in an easily accessible format.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Peltzer et al
    
  </div>

  <div class="pub-publication">
    
    In <em>Genome Biology</em>,
    
    2016
  </div>

  <div class="pub-links">
    
















<a class="btn btn-primary btn-outline btn-xs" href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0918-z" target="_blank" rel="noopener">
  Source Document
</a>



  </div>

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-md-12 isotope-item pubtype-2 year-2014">
          
            <div class="pub-list-item card-simple" itemscope itemtype="http://schema.org/CreativeWork">

  
  

  <h3 class="article-title" itemprop="name">
    <a href="https://apeltzer.github.io/publication/01-inphap/" itemprop="url">inPHAP: Interactive visualization of genotype and phased haplotype data</a>
  </h3>

  <div class="pub-abstract" itemprop="text">
    
    To understand individual genomes it is necessary to look at the variations that lead to changes in phenotype and possibly to disease. However, genotype information alone is often not sufficient and additional knowledge regarding the phase of the variation is needed to make correct interpretations. Interactive visualizations, that allow the user to explore the data in various ways, can be of great assistance in the process of making well informed decisions. But, currently there is a lack for visualizations that are able to deal with phased haplotype data.
    
  </div>

  <div class="pub-authors" itemprop="author">
    
    Jager et al
    
  </div>

  <div class="pub-publication">
    
    In <em>BMC Bioinformatics</em>,
    
    2014
  </div>

  <div class="pub-links">
    
















<a class="btn btn-primary btn-outline btn-xs" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-200" target="_blank" rel="noopener">
  Source Document
</a>



  </div>

</div>

          
        </div>

        
      </div>

    </div>
  </div>
</div>
<footer class="site-footer">
  <div class="container">

    

    <p class="powered-by">

      &copy; 2018 &middot; 

      Powered by the
      <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
      <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>

      

      
      
    

    <script src="/js/hugo-academic.js"></script>
    

    
    

    
    
    

    
    

    
    
    <script>
      const search_index_filename = "/search.json";
      const i18n = {
        'placeholder': "Search...",
        'no_results': "No results found"
      };
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks"
        };
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    <script src="/js/search.js"></script>
    

    
    

  </body>
</html>

